src/renderers/WebGLRenderer.js

```diff
@@ -2512,6 +2512,12 @@ function WebGLRenderer( parameters ) {
 
 		var framebuffer = properties.get( renderTarget ).__webglFramebuffer;
 
+		if ( activeCubeFaceIndex != undefined ) {
```

Comment:
Should we check if `renderTarget.isWebGLRenderTargetCube`?

Reply:
```suggestion
		if ( renderTarget.isWebGLRenderTargetCube && activeCubeFaceIndex != undefined ) {
```
Yes. In my projects i often use `instanceof` to check an object being of a certain class. I forgot that there are these flags :)